shared/pubsub.js

```diff
@@ -74,6 +74,7 @@ export type UnsubMessage = {
 
 export const NOTIFICATION_TYPE = Object.freeze({
   ENTRY: 'entry',
+  GI_VERSION: 'gi_version',
```

Comment:
Can we make this more generic? The pubsub system should ideally be usable from any app.

Reply:
Defining this type as just a plain string would allow to remove this `GI_VERSION` reference, at the cost of a small loss of Flowtype coverage.

Reply:
I mean, can't it be modified to just `VERSION` without issue?

Reply:
It can, that's what it was initially. Then I remembered what you said about identifiers like `id`, i.e. that they were too generic and many things in the app can have one. I guess many things can have their own version too, e.g. the browser, or js libraries. So I went with GI_VERSION instead.

Reply:
@snowteamer then can you modify it to `APP_VERSION`?